### PR TITLE
cavatica workflow test run

### PR DESCRIPTION
**Pull request by naqvia**
Testing on sample data from CAVATICA pre-processed output files from workflow. This includes: 
```

- test files generation
- gnomad filtering
- ensure that script runs and outputs desired files

```

### CODE REVIEW

**Comment by naqvia** [comment]
I still need to upload test files and update readme, but the script now runs-- here is a sample command run: 
`Rscript 01-annotate_variants_CAVATICA_input.R --vcf input/test-cavatica.single.vqsr.filtered.vep_105.vcf --intervar input/test-cavatica.hg38_multianno.txt.intervar --multianno input/test-cavatica.hg38_multianno.txt --variant_summary input/variant_summary.txt --submission_summary input/submission_summary.txt --autopvs1 input/test-cavatica.autopvs1.tsv`

Some of the variables were not updated which were causing it to crash. May be of interest to have a "check" to see if input files and variables are in correct format or some simple QC step like that since the scripts are super sensitive to format.

**Comment by naqvia** [comment]
Added : ` mutate(gnomad_af     = if_else( as.numeric( str_match(INFO, gnomad_var_to_match)[,2])  > filter_variant_af, "PASS","FAIL")) ` , which will take in any user-inputted gnomad variable and threshold.

**Comment by naqvia** [comment]
this PR should address https://github.com/diskin-lab-chop/AutoGVP/issues/32 https://github.com/diskin-lab-chop/AutoGVP/issues/24 and https://github.com/diskin-lab-chop/AutoGVP/issues/17

File: AutoGVP/01-annotate_variants_CAVATICA_input.R [hunk 1]
```diff
@@ -70,37 +70,39 @@ opt <- parse_args(OptionParser(option_list = option_list))
...
-                  mutate(gnomad_af     = if_else( as.numeric( str_match(INFO, "gnomad_3_1_1_AF_non_cancer\\=(0\\.\\d+)\\;")[,2])  > filter_variant_af, "PASS","FAIL")) %>% 
+                  mutate(gnomad_af     = if_else( as.numeric( str_match(INFO, gnomad_var_to_match)[,2])  > filter_variant_af, "PASS","FAIL")) %>% 
                   mutate(variant_af    = if_else(as.integer(str_match(Sample, ":(\\d+)\\,(\\d+)") [,3]) / ( (as.integer(str_match(Sample, ":(\\d+)\\,(\\d+)") [,2]) ) + as.integer(str_match(Sample, ":(\\d+)\\,(\\d+)") [,3] )) > filter_variant_af, "PASS", "FAIL"))
   return(clinvar_vcf)
 }
 
 ## retrieve and store clinVar input file into table data.table::fread()
-vcf_input <-  vroom(input_vcf_file, comment = "#",delim="\t", col_names = c("CHROM","START","ID","REF","ALT","QUAL","FILTER","INFO","FORMAT","Sample"), show_col_types = TRUE)
+vcf_input <-  vroom("/Users/naqvia/Documents/GitHub/pathogenicity-assessment/AutoGVP/input/test-cavatica.single.vqsr.filtered.vep_105.vcf", comment = "#",delim="\t", col_names = c("CHROM","START","ID","REF","ALT","QUAL","FILTER","INFO","FORMAT","Sample"), show_col_types = TRUE)
```
Comment (rjcorb) [inline AutoGVP/01-annotate_variants_CAVATICA_input.R:105]:
```suggestion
vcf_input <-  vroom(input_clinVar_file, comment = "#",delim="\t", col_names = c("CHROM","START","ID","REF","ALT","QUAL","FILTER","INFO","FORMAT","Sample"), show_col_types = TRUE)
```

**Review by rjcorb — CHANGES_REQUESTED** [review]
input vcf variable file needs changing (see suggestion), otherwise runs fine.

**Pull request merged**